reo/tests/test_reopt_url.py

```diff
@@ -118,12 +117,12 @@ def test_valid_data_types(self):
         validator = ValidateNestedInput(self.complete_valid_nestedpost)
 
         for attribute, test_data in validator.test_data('type'):
-
-            response = self.get_response(test_data)
-            text = "Could not convert " + attribute
-            err_msg = str(json.loads(response.content)['messages']['input_errors'])
-            self.assertTrue(text in err_msg)
-            self.assertTrue("(OOPS)" in err_msg)
+            if attribute not in ["doe_reference_name", "percent_share"]:
```

Comment:
this `if` should be removed, right @sakshi-mishra?

Reply:
Just noting that from our recent conversation, this has a history of being present in the public branch and we're keeping it in for now.

Reply:
@sakshi-mishra disregard: we are keeping it as is since it is also on the private master branch (I thought that we had removed it).